src/App.tsx

```diff
@@ -1,14 +1,55 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect, useCallback } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+import { SortType } from './types/sortType';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [isLoaded, setIsLoaded] = useState(false);
+  const [query, setQuery] = useState('');
+  const [sortType, setSortType] = useState<SortType | string>(SortType.ALL);
+
+  const loadedUserFromServer = useCallback(async () => {
+    try {
+      const UserFromServer = await getTodos();
+
+      setTodos(UserFromServer);
+      setIsLoaded(true);
+    } catch (error) {
+      throw new Error('Error. Can`t upload todos');
```

Comment:
if you receive an error while processing todos loading will be better to handle it via separate variable in state